lib/internal/Magento/Framework/File/Uploader.php

```diff
@@ -611,7 +611,7 @@ public static function getNewFileName($destinationFile)
      * @param string $fileName
      * @return string
      */
-    public static function getDispretionPath($fileName)
+    public static function getDispersionPath($fileName)
```

Comment:
Renaming public methods is backward incompatible. 
Please, refactor this codes with following steps: 

- introduce new method **getDispersionPath**
- move logic from old **getDispretionPath** method to created **getDispersionPath**
- mark old method as @deprecated
- make proxy call from old method to new

```
public static function getDispretionPath($fileName)
{
      return self::getDispersionPath($fileName);
}
```

Backward compatible development rules described in http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/

Reply:
You're right, I was too quick. Made the necessary changes